WalletWasabi/Helpers/IoHelpers.cs

```diff
@@ -72,14 +75,14 @@ public static async Task BetterExtractZipToDirectoryAsync(string src, string des
 		public static void EnsureContainingDirectoryExists(string fileNameOrPath)
 		{
 			string fullPath = Path.GetFullPath(fileNameOrPath); // No matter if relative or absolute path is given to this.
-			string dir = Path.GetDirectoryName(fullPath);
+			string? dir = Path.GetDirectoryName(fullPath);
```

Comment:
This is strange. The string can be null. Is this something to do with Nullable?

Reply:
https://source.dot.net/#System.Private.CoreLib/Path.cs,f9128b02ffd0d3ea

Reply:
ACK